CHANGELOG.md

```diff
@@ -1471,3 +1471,22 @@ This is a full list of proofs that have changed form to use irrelevant instance
   1/neg⇒neg : ∀ p .{{_ : NonZero p}} → (1/p : Negative (1/ p)) → Negative p
   ```
 
+* In `Data.List.NonEmpty.Base`:
+  ```agda
+  drop : ℕ → List⁺ A → List⁺ A
+  ```
+  When droping more than the size of the length of the list, the last element remains.
+  
+* Added new proofs in `Data.List.NonEmpty.Properties`:
+  ```agda
+  length-++⁺ : (l : List A) (l′ : List⁺ A) → length (l ++⁺ l′) ≡ List.length l + length l′
```

Comment:
We generally drop the individual arguments in the changelog, just give people the "flavour of the type" rather than the full thing which is usually pretty unreadable.
e.g.
```
 length-++⁺ : length (l ++⁺ l′) ≡ length l + length l′
 ++⁺-cancelˡ′ : l ++⁺ l″ ≡ l′ ++⁺ l‴ → length l ≡ length l′ → l″ ≡ l‴
```

Reply:
done